controllers/cards.js

```diff
@@ -1,36 +1,48 @@
+const mongoose = require('mongoose');
 const Card = require('../models/card');
 
+const { ObjectId } = mongoose.Types;
+
+const NotFoundError = require('../errors/error_not_found');
+const Error500 = require('../errors/error_500');
+
 module.exports.getAllCards = (req, res) => {
   Card.find({})
     .then((card) => {
       if (card.length === 0) {
-        return res.status(404).send({ message: 'База данных cards пуста! ' });
+        throw new NotFoundError('База данных карточек пуста!');
       }
       return res.send({ data: card });
     })
-    .catch((error) => res.status(500).send({ message: error.message }));
+    .catch(new Error500('Ошибка сервера'));
```

Comment:
Нужно исправить: ошибка не обрабатывается централизованным обработчиком.
Централизованная обработка ошибок запустится только если будет вызван `next`:
```suggestion
    .catch(next);
```

Только нужно не забыть объявить `next` здесь:
https://github.com/MaximMaslovVik/15_sprint/blob/9aae875d58b5031fbc6c13ac1f5332065778c271/controllers/users.js#L13